### PR TITLE
GVA: Response Text UI

### PR DESCRIPTION
**Pull request by AndriiHorishniiMOC**
Added Response Text view holder. Used the operator message view for the new view holder.

[MOB-2364](https://glia.atlassian.net/browse/MOB-2364)

[MOB-2364]: https://glia.atlassian.net/browse/MOB-2364?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/chat/adapter/ChatAdapter.kt [hunk 1]
```diff
@@ -151,13 +152,14 @@ internal class ChatAdapter(
...
-                SystemMessageViewHolder(
+                val operatorMessageBinding = ChatOperatorMessageLayoutBinding.inflate(inflater, parent, false)
+                GvaResponseTextViewHolder(
+                    operatorMessageBinding,
                     ChatReceiveMessageContentBinding.inflate(
                         inflater,
-                        parent,
-                        false
+                        operatorMessageBinding.contentLayout,
+                        true
```
Comment (DavDo) [inline widgetssdk/src/main/java/com/glia/widgets/chat/adapter/ChatAdapter.kt:162]: as far as I know, the `attachToRoot` should be false for RecyclerView.Adapter

**Review by DavDo — APPROVED** [review]
LGTM, just 2 small suggestions.

File: widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/GvaResponseTextViewHolder.kt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            operatorMessageBinding.chatHeadView.showProfileImage(item.operatorProfileImgUrl)
+        } else {
+            operatorMessageBinding.chatHeadView.showPlaceholder()
+        }
+    }
+
+    private fun updateMessageContentView(item: GvaResponseText) {
+        messageContentBinding.root.text = item.content.fromHtml()
+
+        if (!TextUtils.isEmpty(item.operatorName)) {
```
[the remark this answers is not in the document]
Comment (AndriiHorishniiMOC) [reply widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/GvaResponseTextViewHolder.kt]: fixed

**Comment by AndriiHorishniiMOC** [comment]
!squash

**Review by Pelkar — APPROVED** [review]
LGTM

**Pull request merged**